e2e/stack.go

```diff
@@ -243,8 +243,9 @@ func (s *Stack) runMonomer(ctx context.Context, env *environment.Env, genesisTim
 	env.DeferErr("close tx db", txdb.Close)
 	mempooldb := dbm.NewMemDB()
 	env.DeferErr("close mempool db", mempooldb.Close)
-	ethstatedb := rawdb.NewMemoryDatabase()
-	env.DeferErr("close eth state db", ethstatedb.Close)
+	ethstatedb := state.NewDatabase(rawdb.NewMemoryDatabase())
+	env.DeferErr("close eth state disk db", ethstatedb.DiskDB().Close)
+	env.DeferErr("close eth state trie db", ethstatedb.TrieDB().Close)
```

Comment:
We should close the rawdb. No need to close individual instances of DBs obtained from the statedb.

something like:
```golang
raw := rawdb.NewMemoryDatabase()
env.DeferErr("close raw db", raw.Close)
ethstatedb := state.NewDatabase(raw)
```